src/campaign/campaign.service.ts

```diff
@@ -20,4 +20,9 @@ export class CampaignService {
     }
     return campaign;
   }
+
+  async getAllCampaigns(user: User): Promise<Campaign[]> {
+    const campaigns = await this.campaignRepository.findAll(user);
+    return campaigns;
```

Comment:
There is no need to create a new variable when we will immediately return it as a return value.